src/tagset.jl

```diff
@@ -57,15 +58,15 @@ end
 function _addtag!(ts::MTagSetStorage, plev::Int, ntags::Int, tag::IntTag)
   plnew = plev
   t = Tag(tag)
-  if length(t) > 0
+  if !isNull(t)
     if isint(t)
       error("""Cannot use a bare integer as a tag.
             If you are looking to set the prime level, use the syntax ("x",1)
             (instead of "x,1")""")
       #plev ≥ 0 && error("You can only make a TagSet with one prime level/integer tag.")
       #plnew = parse(Int,t)
     else
-      ntags = _addtag_ordered!(ts, ntags, tag)  
+      ntags = _addtag_ordered!(ts, ntags,IntSmallString(tag))  
```

Comment:
Why is this constructor call needed? `tag` is already `IntTag = IntSmallString`.

Reply:
Yes, this was a mistake I made when changing other calls to `_addtag_ordered!`. I'll fix it - good catch!